packages/mobile/src/home/CashInBottomSheet.test.tsx

```diff
@@ -28,4 +106,71 @@ describe('CashInBottomSheet', () => {
       isCashIn: true,
     })
   })
+  it('Regular Version is visible when experiment is disabled even if ramp is available', async () => {
+    mockFetch.mockResponse(JSON.stringify([mockRampProvider]))
+    const { getByTestId } = render(
+      <Provider
+        store={createMockStore({
+          app: { rampCashInButtonExpEnabled: false },
+        })}
+      >
+        <CashInBottomSheet />
+      </Provider>
+    )
+    await waitFor(() => expect(getByTestId('cashInBtn')))
+  })
+  it('Regular Version is visible when ramp is unavailable', async () => {
+    mockFetch.mockResponse(JSON.stringify([mockRampProviderUnavailable]))
+    const { getByTestId } = render(
+      <Provider
+        store={createMockStore({
+          app: { rampCashInButtonExpEnabled: false },
```

Comment:
```suggestion
          app: { rampCashInButtonExpEnabled: true },
```
I think this test isn't testing the ramp response right now, just the feature flag (since the feature flag is set to false)

Reply:
though I would imagine we would only ramp the experiment to regions where user has ramp available.